aggregator/map.go

```diff
@@ -57,6 +57,7 @@ const (
 	// nolint: megacheck
 	unknownMetricCategory metricCategory = iota
 	untimedMetric
+	timedMetric
```

Comment:
ULTRA-NIT:
Any reason to change the enum here? I dont think there is any serialization of this value so its not a big deal, but often times things are added to the end of an enum.

Reply:
Yeah there is no serialization involved so the order does not really matter, just that logically timed metric is closer to untimed metric than forward metric, so I'd rather put it next to untimed metric